doc/conf.py

```diff
@@ -18,6 +18,8 @@
 import sphinx
 
 from datetime import datetime
+import warnings
+warnings.filterwarnings('ignore')
```

Comment:
Is this intentional? I don't think we should globally disable warnings for doc builds.

Reply:
That was an overly aggressive `git add -A`, thanks!